### PR TITLE
Add 2.6 to version history

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Add 2.6 to version history

### Checklist
- [x ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by carolxob — APPROVED** [review]
LGTM!

**Review by hdhalter — COMMENTED** [review]
I believe we have 4 experimental features in this release. 2 for ML, one security analytics, and the simple schema feature.

File: version-history.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ permalink: /version-history/
 
 OpenSearch version | Release highlights | Release date  
 :--- | :--- | :--- 
+[2.6.0](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.6.0.md) | Includes simple schema for observability, index management UI enhancements, SigV4 authentication in Dashboards, Security analytics enhancements, sentence transformer models for model-serving framework, search backpressure at the coordinator node level, and adding maps to dashboards. Experimental features include a new ML model health dashboard. For a full list of release highlights, see the Release Notes. | 28 February 2023
```
Comment (hdhalter) [inline version-history.md:13]: Analytics in Security Analytics should be capitalized

**Comment by kolchfa-aws** [comment]
@hdhalter Could you provide a suggestion with what we need to add please? We are also linking to release notes, where all the features should be enumerated.

**Comment by hdhalter** [comment]
I just heard that search backpressure is now production ready.

**Comment by kolchfa-aws** [comment]
Search backpressure was never experimental.

File: version-history.md (hunk 1 above)
Comment (hdhalter) [reply version-history.md:13]:
```suggestion
[2.6.0](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.6.0.md) | Includes simple schema for observability, index management UI enhancements, Security Analytics enhancements, search backpressure at the coordinator node level, and the ability to add maps to dashboards. Experimental features include a new ML model health dashboard, new text embedding models in ML, and SigV4 authentication in Dashboards. For a full list of release highlights, see the Release Notes. | 28 February 2023
```

Comment (Naarcha-AWS) [reply version-history.md:13]: Change the fragment for ML to "...support for sentence transformer pretrained models in ML..."

**Review by Naarcha-AWS — APPROVED** [review]
Small tweak to ML language, but otherwise LGTM

**Pull request merged**